09.11/+(9)Arr_Dyn_2D.cpp

```diff
@@ -0,0 +1,69 @@
+﻿#include <iostream>
+#include <ctime>
+#include <random>
+
+using namespace std;
+
+void findMin(int min, int *arr_2D, int rows, int cols)
+{
+	for (int i = 0; i < rows; i++)
+	{
+		for (int j = 0; j < cols; j++)
+		{
+			if (arr_2D[j] < min)
+				min = arr_2D[j];
+		}
+		cout << "\nMin in this col is " << min << "\t" << endl;
+	}
+}
+
+int main()
+{
+	int rows; //Ввод строк
+	cout << "Enter value of rows in array = " << endl;
+	cin >> rows;
+
+	int cols; //Ввод столбцов
+	cout << "\nEnter valuse of cols in array = " << endl;
+	cin >> cols;
+
+	int **arr_2D = new int*[rows]; //Создание массива и выделение под него памяти
+
+	for (int i = 0; i < rows; i++)
+	{
+		arr_2D[i] = new int[cols];
+	}
+
+	unsigned rand_value = 11;
+	srand(time(0));
+	for (int i = 0; i < rows; i++) //Заполнение массива
+	{
+		for (int j = 0; j < cols; j++)
+		{
+			arr_2D[i][j] = rand() % 50;
+		}
+	}
+
+	for (int i = 0; i < rows; i++) //Вывод массива
+	{
+		for (int j = 0; j < cols; j++)
+		{
+			cout << arr_2D[i][j] << "\t";
+		}
+		cout << endl; 
+	}
+
+	int min = 2147483647;
+
+	findMin(min, *arr_2D, rows, cols);

```

Comment:
int *_arMin = new int[cols];
findMin(arr_2D, rows, cols, _arMin);
    cout << "-------------------------------------------" << endl;
    for (int k =0; k<rows; k++)
    {
        cout << _arMin[k] << endl;
    }